solidity/slither.config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "filter_paths": "contracts/test|node_modules/@keep-network/keep-core",
+    "filter_paths": "contracts/test|node_modules/@keep-network/keep-core|node_modules/@keep-network/sortition-pools",
```

Comment:
I wouldn't ignore sortition pools, at least not for now. There is no separate build just for sortition pools.